parsl/tests/test_docs/test_workflow1.py

```diff
@@ -22,6 +22,7 @@ def save(message, outputs=[]):
     return 'echo {m} &> {o}'.format(m=message, o=outputs[0])
 
 
+@pytest.mark.shared_fs
```

Comment:
I think this one doesn't need a shared filesystem: it is the job of the staging provider to bring the "output.txt" file back from the remote side onto the local filesystem, where it will be visible. (that's what the staging_required marker is indicating) ?

Reply:
Fixed.